client/monaco_editor.js

```diff
@@ -37,6 +37,8 @@ export default class MonacoEditor {
                     language: 'javascript',
                     fontSize: 12,
                     theme: 'vs-dark',
+                    fixedOverflowWidgets: true,
```

Comment:
This is to fix this issue: <https://stackoverflow.com/questions/72143093/monaco-editor-resize-property-causes-editor-popups-to-be-hidden>